### DISCUSSION TITLE
Recommendations SMAD3, phospho-SMAD2 and phospho-TAK1 antibodies human lung FFPE

### DISCUSSION THREAD

**Discussion by Rosa-234** [IBEXImagingCommunity/ibex_imaging_knowledge_base#200] [Q&A] [answered]
We have been trying to optimize stainings for some signaling proteins in human lung FFPE sections.
I was wondering if anyone has recommendations for SMAD3, phospho-SMAD2 and/or phospho-TAK1 antibodies?
We've tried #9513 Cell signaling, #3108 Cell signaling, #28958-1-AP Proteintech and #4536 Cell signaling with no luck so far.
Thank you!

**Comment by radtkea** [comment]
Hi Rosa-234,

Thanks very much for your question. Sorry for my delay in responding. While I haven't looked at those targets in human FFPE samples, I have evaluated BCR signaling _in situ_ using downstream phosphorylation events in IBEX images. This was technically very challenging for several reasons not limited to validating appropriate antibodies and detecting phosphoproteins in large human tumor samples. Dephosphorylation can occur quickly due to existing phosphatases and the slow rate of fixation (1-2 mm/hour). I obtained inconclusive results with several antibody candidates and shared our negative results with the IBEX Imaging Community: [Phospho NF-Kappa B p105](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/supporting_material/Phospho_NF-Kappa_B_p105_Unconjugated/0000-0003-4379-8967.html), [Phospho-Akt](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/supporting_material/Phospho-Akt_Unconjugated/0000-0003-4379-8967.html), and [Phospho-GSK-3alpha](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/supporting_material/Phospho-GSK-3alpha_Unconjugated/0000-0003-4379-8967.html).

[SMAD3 CST #9513](https://www.cellsignal.com/products/primary-antibodies/smad3-antibody/9513): Based on the information on their site, this antibody is validated for Immunofluorescence (IF). If you scroll down and select "Immunofluorescence (Immunocytochemistry)" from the dropdown menu for Protocols, you will see that CST evaluated this antibody in cells fixed with formaldehyde. The example images confirm this by showing images of cells that are either serum-starved (negative control) or treated with hTGF-β3 (positive control). Therefore, this antibody may not work in FFPE samples because of epitope loss following antigen retrieval or other reasons.

[Phospho-SMAD2 CST #3108](https://www.cellsignal.com/products/primary-antibodies/phospho-smad2-ser465-467-138d4-rabbit-mab/3108): This antibody has only been validated for WB (Western Blot) by CST. Therefore, it's perhaps not surprising that this wouldn't work for FFPE. You will want to select for antibodies validated for IHC-P (IHC-Paraffin).

[Phospho-TAK1 Proteintech #28958-1](https://www.ptglab.com/products/Phospho-MAP3K7-Thr187-Antibody-28958-1-AP.htm): The vendor has only validated this antibody for WB. 

[Phospho-TAK1 (Thr187) CST #4536](https://www.cellsignal.com/products/primary-antibodies/phospho-tak1-thr187-antibody/4536): Same as above. Only validated for WB.

[Abcam](https://www.abcam.com/en-us/search?facets.application=IHC-P&sorting=relevance&keywords=smad3) reports several antibodies validated for IHC-P. I would search their catalog, paying close attention to antibodies validated for IHC-P and appropriate antigen retrieval conditions.
![image](https://github.com/user-attachments/assets/820296b4-e4e5-4d14-a748-b35af14b0c45)

[CiteAb](https://www.citeab.com/) is a great resource for finding antibodies as well!

Controls are obviously critical and will require a bit of work to obtain. If possible, I recommend that you create cell suspensions from serum starved and hTGF-β3 treated samples (see CST entry above). Then, prepare cell blocks from these suspensions as described [here](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6628727/). This will mimic the conditions in the tissue and help you with antibody validation.

Best of luck! Please share what you learn along the way. Would be delighted to help you enter the "No" reagents in the Knowledge-Base.

**Reply by Rosa-234** [reply, marked as answer]
Thank you so much! We have ordered some new antibodies from Abcam, hopefully we are more lucky with those.

**Reply by radtkea** [reply]
Wonderful! Best of luck Rosa!
